### DISCUSSION TITLE
Hosting for terra-classic docs

### DISCUSSION THREAD

**Discussion by TheArchitect123** [classic-terra/core#141] [Ideas]
**Problem definition**
Hi @ZaradarBH. Are we still relying on TFL for hosting the classic-docs?
If we need docs, please consider using github pages and hosting a react page for it.

Otherwise I can host it myself on my org.
But tell me what you think first. There needs to be a single official site for Docs.
@edk208

**Comment by ZaradarBH** [comment]
@TheArchitect123 we are aware of the need to migrate the DNS for both the classic docs and agora since they belong to an outside entity. However we are  currently focusing on updating the documentation first, then we will look into DNS & hosting. 

As such it would be epic if you could take charge on this and get a gov prop passed to move the DNS for Agora + classic-docs to another domain, preferably one owned by the TGF, and then we can facilitate the infrastructure on your host if you have the bandwidth to support the traffic and dont mind keeping it alive :) (very generous of you btw)

**Reply by ZaradarBH** [reply]
Converted issue to a discussion until its specified and agreed upon :)

**Comment by TheArchitect123** [comment]
@ZaradarBH @edk208 Here's the link to the new terra docs site.
If you guys need any documentation added, please message me and I'll update the site.

https://terradocs.money/
